### ISSUE TITLE
`autoGlobalExportsFiles` options does not work

### ISSUE THREAD

**Issue by armaan-kumar** [fossamagna/gas-webpack-plugin#879]
I am trying to use this template: https://github.com/iansan5653/gas-ts-template
But exports in build/index.js after typescript compilation do not expose to global object even after adding path in `autoGlobalExportsFiles` array.

**Comment by iansan5653** [comment]
I am able to reproduce this as well and it appears to be the source of https://github.com/iansan5653/gas-ts-template/issues/2

Simple declaration exports do not work:

```ts
// ❌ Does NOT work
export function one() { /* ... */ }

export const two = () => { /* ... */ }
```

But object exports and re-exports from other files do:

```ts
// ✅ Does work:
function three() { /* ... */ }

export {three}

export {four} from "./four.ts"
```

**Comment by fossamagna** [comment]
@armaan-kumar @iansan5653 
I published gas-webpack-plulgin@2.6.0. I added to support to generate  from function declaration exports with autoGlobalExportsFiles.
